<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectBookmark</class>
 <widget class="QWidget" name="KWSelectBookmark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>219</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="bookmarkList"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonRename">
     <property name="text">
      <string>&amp;Rename</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonDelete">
     <property name="text">
      <string>&amp;Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>152</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
